src/client/components/Graph.vue

```diff
@@ -114,15 +107,26 @@ onMounted(() => {
     <div ref="container" h-100vh w-full />
     <div
       border="~ main"
-      absolute bottom-3 right-3 z-100 rounded px3 py1 shadow bg-main
-      flex="~ gap-2"
+      absolute bottom-3 right-3 z-100 rounded px3 py3 shadow bg-main
     >
-      <span text-sm op50>weight by</span>
-      <RadioGroup
-        v-model="graphWeightMode"
-        name="weight"
-        :options="weightItems"
-      />
+      <div flex="~ gap-2" mb-2>
+        <span text-sm op50>color of</span>
+        <Badge v-for="color of colors" :key="color.type" :class="[`text-[#fff] ${{ vue: 'bg-vue', json: 'bg-json', ts: 'bg-ts', js: 'bg-js', css: 'bg-css', html: 'bg-html', svelte: 'bg-svelte', jsx: 'bg-jsx', tsx: 'bg-tsx' }[color.type]}`]">
```

Comment:
BTW, is there a better way to inject a dynamic color class

Reply:
Just use `:style="{ color }"`, man. Don't go over engineering 😂

Reply:
Sorry bro, I am new to vue&unocss. I need more practice. :(